library.html

```diff
@@ -30,7 +30,12 @@
 					</ul>
 				</div>
 				
-				<center><p>navigation bar/burger nav</p></center>
```

Comment:
I have done the same here on this page. I have added a 'logo' placeholder which will eventually be changed to the music database's name.